src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -1494,23 +1498,27 @@ impl WindTurbine {
 
         // Simple model. stow pos sin simulates the angle of the blades vs wind while deploying
         let air_speed_torque = cur_alpha.to_radians().sin()
-            * (indicated_speed.get::<knot>() * indicated_speed.get::<knot>() / 100.)
+            * (indicated_speed.get::<knot>()
+                * indicated_speed.get::<knot>()
+                * Self::AIR_LIFT_COEFFICIENT)
             * 0.5
             * (std::f64::consts::PI / 2. * stow_pos).sin();
+
         self.torque_sum += air_speed_torque;
     }
 
-    fn update_friction_torque(&mut self, displacement_ratio: f64) {
+    fn update_friction_torque(&mut self, displacement: f64, pressure: Pressure) {
```

Comment:
**suggestion**: Probably you can use `Volume` here instead. Bubble this change up.
```suggestion
    fn update_friction_torque(&mut self, displacement: Volume, pressure: Pressure) {
```